CalibCalorimetry/EcalLaserAnalyzer/plugins/EcalStatusAnalyzer.h

```diff
@@ -4,18 +4,20 @@
 #include <map>
 
 #include <memory>
-#include <FWCore/Framework/interface/EDAnalyzer.h>
+#include <FWCore/Framework/interface/stream/EDAnalyzer.h>
```

Comment:
`EDAnalyzers` are typically `global` or `one` since they usually need to see all events.

Reply:
Changed to a `one` module